### ISSUE TITLE
Make stop with error about compiling erim_xml

### ISSUE THREAD

**Issue by yar0d** [erocci/erocci#1]
Hello,
On debian 8 (Linux 3.16.0-4-amd64), I cannot compile erroci, make stop with the following error:
```sh
make[1]: Entering directory '/home/ . . . /erocci/_build/default/lib/erim_xml/priv'
make[1]: *** No rule to make target '/home/ . . . /erocci/_build/default/lib/src/erim_known_attrs.erl', needed by 'All'.  Stop.
make[1]: Leaving directory '/home/ . . . /erocci/_build/default/lib/erim_xml/priv'
===> Hook for compile failed!
Makefile:17: recipe for target 'compile' failed
```
I've tried with and without option "Build erim_xml [Y/n]".

Is there a workaround or fix?

Thank you.

**Comment by yar0d** [comment]
Issue solved with abab632 :-)

**Comment by jeanparpaillon** [comment]
:)
